### PR TITLE
v1.8 -> v1.9

### PR DESCRIPTION
**Pull request by benjaminrwilson**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by izahn** [comment]
@conda-forge-admin please rerender

**Comment by benjaminrwilson** [comment]
@isuruf, it appears that some of the Windows builds are immediately failing. Do you think we can just skip these?

**Comment by benjaminrwilson** [comment]
@hmaarrfk, would you mind taking a look at this when you get a chance as well?

**Comment by benjaminrwilson** [comment]
@conda-forge/core, what is the best practice here for circumventing the Azure CI timeout? Is the only option to build locally?

**Comment by hmaarrfk** [comment]
Currently yes. We have been buliding these packages manually on personally hardware. Rather tedious but dooable for linux.

Did you get the recipe updated? Seems like things are passing. Encouraging!

**Comment by benjaminrwilson** [comment]
Ah, I see. Yes, the recipe is updated and seems to be working. I wonder if we could limit gpu architectures that this is being built against to try and get within the timeout limit.

File: recipe/meta.yaml
Comment (isuruf) [inline recipe/meta.yaml:59]: Why?

Comment (benjaminrwilson) [reply recipe/meta.yaml:59]: I'll remove these. This was an artifact before I added a skip to the windows build.

**Comment by benjaminrwilson** [comment]
@hmaarrfk and @isuruf, who previously built these locally? Do you think they'd be willing to help us out?

**Comment by hmaarrfk** [comment]
I wonder if we should wait for the MKL migration as well. I think the bot is going to be doing an other round and starting a build soon. Do you want me to test building a CUDA build?

**Comment by benjaminrwilson** [comment]
@hmaarrfk, a CUDA build would be much appreciated! I wonder if there's a way to split up the compute architecture builds into separate jobs (to try to avoid the CI timeout)?

**Comment by hmaarrfk** [comment]
I might not have time to start one until Sunday Afternoon. For what its worth, you should be able to test local hardware with `python build-locally.py`

**Comment by hmaarrfk** [comment]
My machiens are just currently in use.

**Comment by hmaarrfk** [comment]
I'm kinda happy with this as is now.

A GPU build
https://anaconda.org/ramonaoptics/pytorch-gpu
https://anaconda.org/ramonaoptics/pytorch

Was successful.

@isuruf do we want to build them all here before merging?

File: recipe/meta.yaml
Comment (isuruf) [inline recipe/meta.yaml:138]: Please remove these 3 lines.

File: recipe/build_pytorch.sh
Comment (isuruf) [inline recipe/build_pytorch.sh:92]: Can you keep this in sync with https://github.com/pytorch/builder/blob/30414521a665e04939a75e42da2aa4908be2fcd6/conda/pytorch-nightly/build.sh#L39 ?

File: recipe/meta.yaml
Comment (benjaminrwilson) [reply recipe/meta.yaml:138]: > For recipe:
 It looks like the 'pytorch-cpu' output doesn't have any tests.

Just curious, why does the linter flag this?

File: recipe/build_pytorch.sh
Comment (benjaminrwilson) [reply recipe/build_pytorch.sh]: Done

**Comment by hmaarrfk** [comment]
@isuruf thank you for the review. I started a build tracker.

For those that want to help chose a build, and run the command:

```
python build-locally.py CONFIG_IN_THE_TABLE
```

and fill in the table with your name and start time (just so that we can get a feeling of the expected end date).

I started 4 at time on a largeish machine i have. I hope that the machine can handle it.

**Comment by hmaarrfk** [comment]
nvm, on my computer, the disk is full..... managing the space taken by docker is no fun....

**Comment by isuruf** [comment]
Please wait until the review is over and this Pr is merged

**Comment by hmaarrfk** [comment]
ok, will do.

File: recipe/meta.yaml
Comment (isuruf) [reply recipe/meta.yaml:138]: Linter doesn't understand selectors. It's wrong to have this here. Please remove them.

Comment (benjaminrwilson) [reply recipe/meta.yaml]: Done

Comment (isuruf) [reply recipe/meta.yaml]: Thanks. FYI, the two lines `- name: pytorch-cpu` and `- name: pytorch-gpu` are mutually exclusive and the lines below them should apply to both.

**Comment by benjaminrwilson** [comment]
@hmaarrfk and @isuruf, thanks for the reviews. Let me know if you have any other concerns.

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Review by isuruf — APPROVED** [review]

**Pull request merged**
